doc/source/conf.py

```diff
@@ -360,3 +357,80 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'https://docs.python.org/3/': None}
+
+
+def rstjinja(app, docname, source):
```

Comment:
It seems this could be refactored as in: https://github.com/pcdshub/pcdsdevices/tree/master/docs/source/_templates/autosummary

With your approval, I'd like to push this task to another PR. The templates would remain the same, just this block of code would be nuked and things would be rearranged.

Reply:
I approve. Issue here: https://github.com/caproto/caproto/issues/582